gateway-service/pom.xml

```diff
@@ -11,10 +11,48 @@
 
     <artifactId>gateway-service</artifactId>
 
-    <properties>
```

Comment:
For all: let's try to remove these properties.
I think we can declare them just once at the parent level